src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs

```diff
@@ -100,9 +102,11 @@ public void Configure(string name, OpenIdConnectOptions options)
         private async Task<OpenIdClientSettings> GetClientSettingsAsync()
         {
             var settings = await _clientService.GetSettingsAsync();
-            if ((await _clientService.ValidateSettingsAsync(settings)).Any(result => result != ValidationResult.Success))
+            var errorResult = (await _clientService.ValidateSettingsAsync(settings)).Where(r => r != ValidationResult.Success);
+
+            if (errorResult.Any())
             {
-                _logger.LogWarning("The OpenID Connect module is not correctly configured.");
+                _logger.LogError($"The OpenId Connect module is not correctly configure : { string.Join(", ", errorResult.Select(r => r.ErrorMessage))}" );
```

Comment:
```suggestion
                _logger.LogWarning("The OpenID Connect module is not correctly configured : {Errors}", string.Join(", ", errors.Select(e => e.ErrorMessage)));
```

Reply:
If you write like this, will the result of the following calculation be filled into the {Errors} expression?